go/logictest/resultparser.go

```diff
@@ -90,6 +91,18 @@ func parseLogEntry(scanner *parser.LineScanner) (*ResultLogEntry, error) {
 			continue
 		}
 
+		secondSpace := strings.Index(line[firstSpace+1:], " ") + firstSpace
+		if secondSpace == -1 {
```

Comment:
This will never be true, since you're adding firstSpace. You need to compare the result of .Index directly